org.eclipse.acute/src/org/eclipse/acute/dotnetnew/DotnetNewWizard.java

```diff
@@ -34,8 +35,16 @@
 	private DotnetNewWizardPage wizardPage;
 	@Override
 	public void init(IWorkbench workbench, IStructuredSelection selection) {
-		// TODO
 		wizardPage = new DotnetNewWizardPage();
+
+		Object firstElement = selection.getFirstElement();
+		if (firstElement instanceof IResource) {
```

Comment:
We should probably handle IWorkingSet too (same thing, it can be an adapter to IWorkingSet). So it one selects a working set, this ws is pre-set as the target one in the working set area.
If no working set is selected, but we have an IResource, the working set part of the dialog should be initiated with the working set of the IProject containing the IResource.